src/components/flyout/Flyout.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { select } from '@storybook/addon-knobs';
+
+import Button from '../button';
+import IconHelp from '../../icons/general/IconHelp';
+// @ts-ignore JS import
+import TextInput from '../text-input';
+import PlainButton from '../plain-button';
+import PrimaryButton from '../primary-button';
+// @ts-ignore JS import
+import TextArea from '../text-area';
+
+// @ts-ignore JS import
```

Comment:
Check if we need these lines in the code or not, if you dont see any errors or warnings, remove them

Reply:
There is an error that looks like this
<img width="1202" alt="image" src="https://user-images.githubusercontent.com/34158284/182713617-4f248a36-f25c-41a3-8515-3971cb5d1cab.png">